database/migrations/2021_04_25_193708_add_columns_to_plasma_donors_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddColumnsToPlasmaDonorsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('plasma_donors', function (Blueprint $table) {
+            $table->tinyInteger('verified')->default(0);
```

Comment:
should be `$table->boolean`.

Boolean also implements tinyInteger, the only major difference is boolean assigns a size of tinyInteger(1)